xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-api/src/main/java/org/xwiki/ratings/script/RatingsScriptService.java

```diff
@@ -397,4 +411,42 @@ public AverageRatingApi getUserReputation(DocumentReference username)
             return null;
         }
     }
+
+    /**
+     * Retrieves the lowest level ratings configuration document in the hierarchy and fallback to XWiki.RatingsConfig
```

Comment:
I think the javadoc would provide a better explanation, it's a bit hard to understand ATM